services-manager/README.md

```diff
@@ -37,142 +37,64 @@ The Palisade Services Manager is a configuration-driven process-spawner and REST
 
 The manager is designed to be used by defining a collection of SpringBoot configuration files:
  * One set for the manager itself - this defines how it should start up further services or reconfigure those already running
-    * These are located in the service-manager's `resources` directory
  * One set for each service - these are discovered through the above configuration and passed on as an argument to the service jar
-    * These are located in each service's `resources` directory
+    * These wll all already be located in each service's `resources` directory
  
-The manager works in several modes, operated by the use of several flags:
- * `--manager.mode=run` - run the *schedule*, in turn running a sequence of *tasks*, with each task running a collection of *services*
- * `--manager.mode=shutdown` - shutdown the services in the opposite order to how they were started in the schedule
- * `--manager.mode=loggers` - perform a change to the logging level of configured services by REST requests (POST /actuator/loggers/*)
```

Comment:
Big bad merge
Run `mvn install -pl services-manager` to generate this file, or just revert the changes